content/cli/users.md

```diff
@@ -46,8 +46,7 @@ Create a new user on Cloudify Manager.
 
 #### Optional flags
   
-* `-r, --security-role [admin|user]` - A role to specifies the user's permissions
-                                  on the manager. (default: default)
+* `-r, --security-role [sys_admin|default]` - A role to specify if the user is a sys-admin on the manager (in which case, the  role 'sys_admin' should be provided). If not, the user will be created as 'default', meaning you will needs to be explicitly assign it with tenants to access their resources. (default: default)
```

Comment:
A role that defines the user as a 'sys-admin' (admin user) or 'default' (non-admin user). A 'default' user must be explicitly assigned to tenants in order to perform actions and access resources.